src/sentry/static/sentry/app/components/activity/note.jsx

```diff
@@ -6,7 +6,52 @@ import ConfigStore from '../../stores/configStore';
 import LinkWithConfirmation from '../../components/linkWithConfirmation';
 import {t} from '../../locale';
 
+function isSafeHref(href) {
+  try {
+    return /^https?:\/\/./i.test(decodeURIComponent(unescape(href)));
+  } catch(e) {
+    return false;
+  }
+}
+
+
+// We need to implement our own marked Renderer to not render
+// potentially malicious uris.
+// This is copy/pasted from
+// https://github.com/chjj/marked/blob/master/lib/marked.js#L869-L888
+// and modified.
+function Renderer(options) {
+  this.options = options || {};
+}
+
+Object.assign(Renderer.prototype, marked.Renderer.prototype);
+
+Renderer.prototype.link = function(href, title, text) {
+  // For a bad link, just return the plain text href
+  if (this.options.sanitize && !isSafeHref(href)) return href;
```

Comment:
Can we really make this assumption that returning `href` is safe? I'd prefer to mirror the Marked behavior. Also, this means users can't use relative paths to point to URLs inside of Sentry.

Why not just do:

```
Render.prototype.link = function (href, title, text) {
  if (this.options.sanitize && !isSafeHref(href))
    return '';
  
  return marked.Renderer.prototype.apply(this, arguments);
}
```

... and same for image. Seems like it'll stand up to Marked changes better.